org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/SwitchCase.java

```diff
@@ -122,7 +122,7 @@ public static List propertyDescriptors(int apiLevel) {
 	/**
 	 * Indicates whether <code>optionalExpression</code> has been initialized.
 	 */
-	private boolean expressionInitialized = false;
+	private volatile boolean expressionInitialized = false;
```

Comment:
`= false` could be also removed

Reply:
isn't there a cleanup for remove unnecessary assignment of default value? If not maybe @jjohnstn could implement one?